src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -181,6 +181,14 @@ type MonoConfig = {
     extensions?: {
         [name: string]: any;
     };
+    /**
+     * This is current working directory for the runtime on the virtual file system. Default is "/".
```

Comment:
current or initial at startup? how do we keep it in sync if current?

Reply:
initial